ql/src/java/org/apache/hadoop/hive/ql/parse/CalcitePlanner.java

```diff
@@ -2296,6 +2297,7 @@ private RelNode applyPostJoinOrderingTransform(RelNode basePlan, RelMetadataProv
         rules.add(JDBCFilterJoinRule.INSTANCE);
         rules.add(JDBCFilterPushDownRule.INSTANCE);
         rules.add(JDBCProjectPushDownRule.INSTANCE);
+        rules.add(JdbcAggregateProjectMergeRule.INSTANCE);
```

Comment:
Should it be called `JBDCAggregateProjectMergeRule` to follow the convention?

Reply:
done!